src/test/scala/org/hammerlab/guacamole/assembly/DeBrujinGraphSuite.scala

```diff
@@ -38,15 +40,67 @@ class DeBruijnGraphSuite extends GuacFunSuite {
 
   test("build graph with short kmers and correct counts") {
 
-    val sequence = Bases.stringToBases("TCATCTTAAAAGACATAAA")
+    val sequence = makeRead("TCATCTTAAAAGACATAAA")
     val graph = DeBruijnGraph(Seq(sequence), kmerSize = 3)
 
     graph.kmerCounts === Map("TCA" -> 1, "CAT" -> 2, "AAA" -> 3)
   }
 
+  test("build graph and prune sequence based on kmer quality") {
+
+    val highQualitySequence = makeRead(
+      "TCATCTTAAAAGACATAAA",
+      qualityScores = Some((0 until 19).map(_ => 30)) // All base qualities are 30
```

Comment:
nit: I like 2 spaces before inline line-comments; again, lmk if you agree and I'll add to #439